tesseract_collision/core/include/tesseract_collision/core/types.h

```diff
@@ -290,6 +290,46 @@ class ContactResultMap
   bool operator==(const ContactResultMap& rhs) const;
   bool operator!=(const ContactResultMap& rhs) const;
 
+  /** @brief Get a brief summary of the most frequently colliding link pair
+   * @return A string containing the collision summary
+   */
+  std::string getCollisionSummary() const
```

Comment:
I think we also just name this `getSummary()`.

Reply:
Good point, I don't know what other summary you'd expect from a `ContactResultMap`